.gitignore

```diff
@@ -14,3 +14,4 @@ vendor
 *.orig
 *.diff
 *.sw?
+Gopkg.lock
```

Comment:
Don't add the Gopkg.lock file to .gitignore.  Revert the Gopkg.lock file back to the one found on master and then just don't commit the changes that get made to it.